### PR TITLE
fix(pinned): move some things around and add styling to pinned messages

### LINKED ISSUE #644: bug(Pinned): Buttons for Pinned messages are huge
**Issue:**
The buttons for the Pinned messages modal are huge and should be a smaller size 

**Screenshot:**
image

### PR DESCRIPTION
**Pull request by Jekrimo**
### What this PR does 📖

- move some things around and add styling to pinned messages

### Which issue(s) this PR fixes 🔨

- Resolve #644

### CODE REVIEW

**Comment by stavares843** [comment]
Captura de ecrã 2024-10-01, às 10 14 54


its adding this warning when doing npm run build

**Review by phillsatellite — APPROVED** [review]

**Comment by luisecm** [comment]
@Jekrimo I see the automated test for pinned messages is failing because in this branch we are no longer showing the status indicator in the profile picture/identicon shown next to each pinned message. Is this expected to be removed?

**Comment by Jekrimo** [comment]
Hey @luisecm ! I'm a little confused by what's expected by the tests. I thought the test was failing because of unused css classes, so I removed and then the failed test didn't show up so I thought that was what was needed? Or maybe I'm missin another message? There is a noIndicator tag on the pfp in pinned message as that shouldn't be showin the loading/typing indicator as it would be hidden behind the button but that's a built property so I don't know why that would fail. 

Sorry wall of text haha thanks for your help and happy monday!

File: src/lib/enums/icons.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 const svgOptions = `stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"`
 
 export const enum Shape {
+    GoTO = `<path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M3 4.5h14.25M3 9h9.75M3 13.5h5.25m5.25-.75L17.25 9m0 0L21 12.75M17.25 9v12" />`,
```
Comment (lgmarchi) [inline src/lib/enums/icons.ts]: GoTO should be GoTo?

**Comment by luisecm** [comment]
@Jekrimo you are correct the tests are using the class to validate the user status on the profile picture shown on pinned message. I see that you added this back recently since the test is no longer failing :D

**Review by lgmarchi — APPROVED** [review]

**Pull request merged**